.github/scripts/tests/build.sh

```diff
@@ -4,6 +4,13 @@ source `dirname ${BASH_SOURCE[0]}`/../config.sh
 
 echo "::group::Build Aarch64 tests"
     cd $ROOT_PATH/tests
-    cmake -S . -B build
+
+    if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "build/Makefile" ]]; then
```

Comment:
Nitpick: The "configure" group should be outside of the "build" group to have proper formatting in GitHub Actions logs.

Reply:
Changed